test/e2e/browser_console.feature

```diff
@@ -0,0 +1,117 @@
+  In order to use Karma
+  As a person who wants to write great tests
+  I want to be able to customize how the browser console is logged.
+
+  Scenario: Execute logging program
+    Given a configuration with:
+      """
+      files = ['browser-console/log.js', 'browser-console/test.js'];
+      browsers = ['PhantomJS'];
+      plugins = [
+        'karma-jasmine',
+        'karma-phantomjs-launcher'
+      ];
+      browserConsoleLogOptions = {
+        path: 'console.log',
+        format: '%t:%m'
+     };
+      """
+    When I start Karma
+    Then the file at console.log contains:
+      """
+      log:'foo'
+      debug:'bar'
+      info:'baz'
+      warn:'foobar'
+      error:'barbaz'
+      """
+
+  Scenario: Execute logging program with different format
+    Given a configuration with:
+      """
+      files = ['browser-console/log.js', 'browser-console/test.js'];
+      browsers = ['PhantomJS'];
+      plugins = [
+        'karma-jasmine',
+        'karma-phantomjs-launcher'
+      ];
+      browserConsoleLogOptions = {
+        path: 'console.log',
+        format: '%t:%T:%m'
+      };
+      """
+    When I start Karma
+    Then the file at console.log contains:
+      """
+      log:LOG:'foo'
+      debug:DEBUG:'bar'
+      info:INFO:'baz'
+      warn:WARN:'foobar'
+      error:ERROR:'barbaz'
+      """
+
+  Scenario: Execute logging program with different log-level
+    Given a configuration with:
+      """
+      files = ['browser-console/log.js', 'browser-console/test.js'];
+      browsers = ['PhantomJS'];
+      plugins = [
+        'karma-jasmine',
+        'karma-phantomjs-launcher'
+      ];
+      browserConsoleLogOptions = {
+        path: 'console.log',
+        format: '%t:%T:%m',
+        level: 'warn'
+      };
+      """
+    When I start Karma
+    Then the file at console.log contains:
+      """
+      log:LOG:'foo'
+      warn:WARN:'foobar'
+      error:ERROR:'barbaz'
+      """
+
+  Scenario: Execute logging program when logging browser
+    Given a configuration with:
+      """
+      files = ['browser-console/log.js', 'browser-console/test.js'];
+      browsers = ['PhantomJS'];
+      plugins = [
+        'karma-jasmine',
+        'karma-phantomjs-launcher'
+      ];
+      browserConsoleLogOptions = {
+        path: 'console.log',
+        format: '%b'
+      };
+      """
+    When I start Karma
+    Then the file at console.log contains:
+      """
+      Phantom
+      """
+
+  Scenario: Execute logging program and disabling terminal
+    Given a configuration with:
+      """
+      files = ['browser-console/log.js', 'browser-console/test.js'];
+      browsers = ['PhantomJS'];
+      plugins = [
+        'karma-jasmine',
+        'karma-phantomjs-launcher'
+      ];
+      browserConsoleLogOptions = {
+        path: 'console.log',
+        format: '%b',
+        terminal: false
+      };
+      """
+    When I start Karma
+    Then it passes with:
+      """
+      .
+      PhantomJS
+      """
```

Comment:
I'll add that. Just to be clear, the format will not effect the output in the console.